Demo/app/test/test_max_findings_limit.py

```diff
@@ -0,0 +1,80 @@
+"""
+Test script for the maximum findings limit feature.
+Tests that submissions with more than 20 findings are rejected properly.
+"""
+import asyncio
+import httpx
+import traceback
+from app.models.finding_input import FindingInput, Finding, Severity
+from app.database.mongodb_handler import mongodb
+
+# API base URL
+BASE_URL = "http://localhost:8004"
+# API key for authentication
+API_KEY = "test-api-key"
```

Comment:
These can also be imported from config.py, the same can be applied in test_process_findings.py.